src/components/SuperDrawer/SuperDrawer.tsx

```diff
@@ -8,6 +8,11 @@ import { useBeamContext } from "src/components/BeamContext";
 import { Css, px } from "src/Css";
 import { useTestIds } from "src/utils";
 
+export enum SuperDrawerWidth {
+  small = 560,
+  normal = 1040
+};
```

Comment:
**request:** Can we create a `/components/SuperDrawer/utils.ts` that has these width definitions in it? I'd rather `SuperDrawerContent` not import from `SuperDrawer`. It is probably fine, but I typically prefer to move these sort of utility constants to a non-component related file. Doing this will help prevent circular dependency issues (not a problem right now as `SuperDrawer` does not import from `SuperDrawerContent`, but more of a general preference)

I'm also wondering if this makes sense as an `enum`? I'm not against it, but I usually see these sort of values as a regular object that maybe uses an `enum` as the "key". Something like:
```
const superDrawerWidthToPixels: Record<SuperDrawerWidth, number> = {
  [SuperDrawerWidth.Small]: 560,
  [SuperDrawerWidth.Normal]: 1040,
};
```
And if we are introducing an enum for these values, then I think it'd also be worth using that `enum` for the props.
```
export interface OpenInDrawerOpts {
  ...
  width?: SuperDrawerWidth;
}
```

(bear with me as I think "out loud" here, please...) So maybe we keep your SuperDrawerWidth enum as is, with the "name = pixels", but change it from "small" and "normal" to "Small" and "Normal" to just stick our conventions of `enum` naming.

Then when a user calls this they'd do something along the lines of:
```
openInDrawer({ width: SuperDrawerWidth.Small }); // a bit more typing than just doing "small", but will immediately translate to the pixel value
```
And in the SuperDrawer we can do something like:
```
const { width = SuperDrawerWidth.Normal, ... } = firstContent ?? {}
```
Thoughts?

Reply:
I prefer the `enum` option over the `Record` option, just for code simplicity.